build/config.js

```diff
@@ -60,6 +60,9 @@ export default ({ context = __dirname, mode = 'development', ...opts }) => {
         performance: {
             hints: false,
         },
+        node: {
+            fs: 'empty',
```

Comment:
Can you give me some more info as to why this is needed? Or, what expects node's `fs` module to be available?

Reply:
![image](https://user-images.githubusercontent.com/31558262/52568378-9b49e880-2e34-11e9-88d2-b12c3b19c99f.png)
Without it, I am getting this webpack build error. A quick google search gave me this solution.